tests/test_isg_gen.py

```diff
@@ -49,11 +51,16 @@ def test_init_input_file():
     assert(all([r0, r1, r2]))
 
 
-@pytest.mark.parametrize("levels", [levels, levelfile])
-def test_read_levels_list(levels):
+def test_read_levels_list():
```

Comment:
this test was separated out since there are different sets of levels being read in from different sources. Previously, the input list and the file matched, so I checked them against the same expected value. But now the inputs are different so two different checks.